db/seeds.rb

```diff
@@ -5,3 +5,8 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+
+20.times do
+  Job.create(position: Faker::Name.title, posting_date: rand(1...10).days.ago, description: Faker::Lorem.sentence(3), location: "#{Faker::Address.city}, #{Faker::Address.state}", company_id: '1', source: Faker::Name.name)
```

Comment:
Line is too long. [221/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.